eng/Versions.props

```diff
@@ -42,7 +42,7 @@
     <_MicrosoftWebWebView2Version>1.0.1210.39</_MicrosoftWebWebView2Version>
     <!-- GLIDE - the android maven artifact in /src/Core/AndroidNative/maui/build.gradle -->
```

Comment:
It doesn’t look like the Java version was updated:

https://github.com/dotnet/maui/blob/4aaa996db6506844836036a5d115f2220cf7cdd6/src/Core/AndroidNative/maui/build.gradle#L29